docs/control-plane.md

```diff
@@ -142,6 +142,7 @@ Conditions provide realtime feedback to users on the underlying state of their d
 | modelUri       | String                                                                                             | URI pointing to Saved Model assets |
 | runtimeVersion | String                                                                                             | Defaults to latest the version of Tensorflow. |
 | resources      | [Resources](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/) | Defaults to requests and limits of 1CPU, 2Gb MEM. |
+| persistentVolumeClaim      | [persistentVolumeClaim](#PersistentVolumeClaim) | Mount a [persistent volume claim (PVC)](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) to get model |
```

Comment:
we should amake a note that pvc is optional and needed for scenarios where s3 is not available, or for latency concerns

Reply:
The part has been removed in new solution, thanks.